src/commands/read.ts

```diff
@@ -61,23 +75,46 @@ export const handler = async (argv: Arguments<Options>): Promise<void> => {
       return;
     }
 
-    const { database: db, ens } = await setupCommand(argv);
+    const setup = await setupCommand(argv);
+    const { database: db, ens } = setup;
```

Comment:
Needed a variable for the setup instead of simple destructuring because instance because the getter for the validator can throw an error at this point, so I only access it later once I'm sure it won't.